release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -202,6 +202,13 @@ For more information, see _Managing machines with the Cluster API_.
 [id="ocp-4-16-nodes_{context}"]
 === Nodes
 
+[id="ocp-4-16-must-gather-directory_{context}"]
+==== Information added to directory when using `oc adm must-gather`
+
+This release adds more information to the must-gather directory when the `oc adm must-gather` command is used. Now, the `oc` binary version and the logs generated by the `oc adm must-gather` are added to the must-gather directory. This helps identify issues that may stem from using a specific verions of `oc`, as well as listing what image was used, and if any data could not be gathered.
```

Comment:
Making a few suggestions - don't feel the need to use every one though.

The second sentence is also passive voice. Instead of saying that they are collected/added, maybe you could make it active - saying that the must-gather command now collects these things.

You could also consider whether any of this information could be better presented in a bulleted list. For example, you could consider it at the beginning for the two items that are now additionally being collected (oc version and must-gather logs). Like "This release now collects the following information when you run the `oc adm must-gather command:` or something like that.

I also think there's room for improvement in the last sentence. It's good to list the things it'll help with, but I think you could either do it more parallel (no "as well as"), or maybe it might be good to break it up into that the oc version helps figure out if there any version specific issues, and collecting the must-gather logs lists the must-gather image that was used and whether there was any data that couldn't be gathered.

```suggestion
This release collects additional information when you run the `oc adm must-gather` command. Now, the OpenShift CLI (`oc`) binary version and the logs generated by the `oc adm must-gather` command are added to the `must-gather` directory. This helps identify issues that might stem from using a specific version of `oc`, as well as listing what image was used, and if any data could not be gathered.
```